<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmLocalSize</class>
 <widget class="QMainWindow" name="frmLocalSize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meshing local size </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbxFaceMeshSize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Mesh size</string>
    </property>
    <widget class="QLabel" name="lblThe_start">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Mesh size</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLocalSize">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>181</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="chxLocal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Local mesh size</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbxFaces">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>321</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Faces</string>
    </property>
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnApply">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
